app/models/team.rb

```diff
@@ -36,4 +36,9 @@ def matches
   def has_multiple_owners?
     owners.length > 1
   end
+
+  def in_event?
+    events.length > 0
```

Comment:
Also: If you want you could add a simple model test for testing that the `in_event?` check works.